### PR TITLE
Game window title will now change to reflect what server you are on

### LINKED ISSUE #3292: Display Server Name In-Game
Players currently have no indication of what their current server is. We might want to display this information somerwhere, maybe on the pause screen.

The server name should be retrieved from ```ServerData.ServerStatus.cs``` and displayed somewhere. ~~Alternatively we could make the in-game name separate from ```ServerData.ServerStatus.cs``` and store it in a ```.txt``` file instead. I'm not sure which option is better.~~ nnaaaah.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
while browsing through older issues i found an issue saying that players can't know what server they are on, this is of course very old as you can see what server you are on when you join a server now with the MOTD window. 
However I've decided to add an extra thing to this so it can be more in parity with SS13 with how detailed some servers' make the game's window title be some times.

### Notes :
At some point we should make this method public and let admins change the title themselves like how admins can do on UnityStation
Works sadly only with windows because Unity is not as advanced as other cooler game engines like Godot.

closes #3292

### CODE REVIEW

File: UnityProject/Assets/Scripts/Util/WindowNameChanger.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+			EventManager.AddHandler(Event.PlayerRejoined, ChangeTile);
+			EventManager.AddHandler(Event.RoundStarted, ChangeTile);
+		}
+
+		private void ChangeTile()
+		{
+			//Get the window handle.
+			var windowPtr = FindWindow(null, Application.productName);
+			//Set the title text using the window handle.
+			var serverName = ServerInfoUI.Instance.ServerName.text != "" ? ServerInfoUI.Instance.ServerName.text : "Nameless Server";
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Util/WindowNameChanger.cs:36]: is there anything that could happen from malicious strings from server with this?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Util/WindowNameChanger.cs:36]: I looked into it and I couldn't find anything. There is probably zero way you can do anything malicious by changing the window's title and even if there is we'd probably solve it when we get to the sandboxing/security development phase and open up stationhub to community servers.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Util/WindowNameChanger.cs:36]: ok cool

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Util/WindowNameChanger.cs:36]: the only issue by looks like is if you spam change it causing Windows to blue screen but doesn't seem likely with this code so as all good

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**